### PR TITLE
Global collectd update

### PR DESCRIPTION
**Pull request by cznewt**

### CODE REVIEW

File: linux/meta/collectd.yml [hunk 1]
```diff
@@ -1,31 +1,31 @@
 plugin:
   linux_network_interface:
     plugin: interface
-    interval: 60
+    execution: local
```
Comment (fpytloun) [inline linux/meta/collectd.yml]: Why don't we make execution local as default?

Comment (simonpasquier) [inline linux/meta/collectd.yml]: Didn't we agree on 'location: remote|local'?

File: linux/files/collectd_df.conf [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+</LoadPlugin>
+
 <Plugin df>
   {%- for fs_type in plugin.get('fs_types', []) %}
   FSType {{ fs_type }}
   {%- endfor %}
-  {%- if plugin.get('ignore_selected', False) %}
-  IgnoreSelected true
-  {%- endif %}
+  IgnoreSelected {{ plugin.get('ignore_selected', true)|lower }}
```
Comment (simonpasquier) [inline linux/files/collectd_df.conf]: s/true/True/?

Also it means that you invert the behavior when 'ignore_selected' isn't defined. Is that ok?

Comment (elemoine) [inline linux/files/collectd_df.conf]: Minor, but wouldn't it make more sense to use

```
plugin.get('ignore_selected', True)|lower
```

or

```
plugin.get('ignore_selected', 'true')|lower
```

?

Just a suggestion…

**Pull request merged**